addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -29,16 +29,17 @@ if (_vehicle isEqualType objNull) then {_vehicle = typeOf _vehicle};
 if (_vehicle == "") exitWith {
     ERROR_1("VehicleType [%1] is empty in ace_rearm_fnc_addVehicleMagazinesToSupply",_string);
 };
+private _turrets = [_vehicle] call FUNC(getAllRearmTurrets);
```

Comment:
it was so interesting for me, so I decided to test ;)
on my computer `[[-1]] + allturrets t` is **twice faster** then function call.